build/sync/plan.yml

```diff
@@ -0,0 +1,49 @@
+checks:
+  - type: repo_is_clean
+    params:
+      repo: template
+  - type: repo_is_clean
+    params:
+      repo: plugin
+actions:
+  - paths:
+      - build
+    actions:
+      - type: overwrite_directory
+        params:
+          create: true
+  - paths:
+      - Makefile
+      - .editorconfig
+    actions:
+      - type: overwrite_file
+        params:
+          create: true
+        conditions:
+          - type: file_unaltered
+            params:
+              repo: plugin
+              reference-repo: template
+  - paths:
+      - go.mod
+      - webapp/webpack.config.js
+      - webapp/tsconfig.json
+      - webapp/.eslintrc.json
+      - webapp/src/index.js
+      - server/configuration.go
+      - server/plugin.go
+      - server/plugin_test.go
+      - public/hello.html
```

Comment:
These file is only useful on generation of a new repo. I'm inclined to drop as they will probably never get an update that should be applied to other repos.